### PR TITLE
Feature basic household ui

### LINKED ISSUE #78: [UI] Set up Basic Structure of ReactUI
Basic Setup includes: 
- [ ] [create-react-app](https://github.com/facebook/create-react-app)
- [ ] UI-component library (e.g [Grommet](https://v2.grommet.io/))
- [ ] charting library (e.g [React-vis](https://github.com/uber/react-vis))

### PR DESCRIPTION
**Pull request by dakingha69**
Closes #78 and adds basic first version of `'Household Stats`.

Also adds an endpoint `/household-stats` to the HHS.

### CODE REVIEW

File: household-server/db-handler.js [hunk 1]
```diff
@@ -47,21 +47,30 @@ module.exports = {
...
   /**
    * Method to read data from the database
-   * @param {String} url URL/URI of the DB
+   * @param {string} url URL/URI of the DB
+   * @param {string} collection Name of collection to read from
+   * @param {Object} filter
    * @returns {Promise} Which either resolves into an Array of objects or rejects an error
    */
-  readAll: url => {
+  readAll: (url, collection, filter = {}) => {
```
Comment (simonhuber95) [inline household-server/db-handler.js]: Filter should be incorporated if we already have it there

File: household-ui/public/manifest.json [hunk 2]
```diff
@@ -0,0 +1,15 @@
+{
+  "short_name": "React App",
+  "name": "Create React App Sample",
```
Comment (simonhuber95) [inline household-ui/public/manifest.json]: We shoud rename it soon

File: household-server/db-handler.js (hunk 1 above)
Comment (dakingha69) [reply household-server/db-handler.js]: will open an issue for that... this is just to have something for tmrw

File: household-ui/public/manifest.json (hunk 2 above)
Comment (dakingha69) [reply household-ui/public/manifest.json]: yes definitely

**Pull request merged**